librespot-android-sink/src/main/java/xyz/gianlu/librespot/android/sink/AndroidSinkOutput.java

```diff
@@ -19,24 +19,51 @@
     private AudioTrack track;
     private float lastVolume = -1;
 
-    public AndroidSinkOutput() {
-    }
-
     @Override
     public boolean start(@NotNull OutputAudioFormat format) throws SinkException {
-        AudioTrack.Builder builder = new AudioTrack.Builder();
-        builder.setAudioFormat(new AudioFormat.Builder()
-                .setSampleRate((int) format.getSampleRate())
-                .build());
+        int pcmEncoding = format.getSampleSizeInBits() == 16 ? AudioFormat.ENCODING_PCM_16BIT : AudioFormat.ENCODING_PCM_FLOAT;
+        int channelConfig = format.getChannels() == 1 ? AudioFormat.CHANNEL_OUT_MONO : AudioFormat.CHANNEL_OUT_STEREO;
```

Comment:
Do we need to expect anything but Stereo or Mono?

Reply:
Those are the only we get and the only ones supported internally.